staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -103,35 +105,35 @@ type waitForDeleteParams struct {
 
 // CheckEvictionSupport uses Discovery API to find out if the server support
 // eviction subresource If support, it will return its groupVersion; Otherwise,
-// it will return an empty string
-func CheckEvictionSupport(clientset kubernetes.Interface) (string, error) {
+// it will return an empty GroupVersion
+func CheckEvictionSupport(clientset kubernetes.Interface) (schema.GroupVersion, error) {
 	discoveryClient := clientset.Discovery()
 	groupList, err := discoveryClient.ServerGroups()
 	if err != nil {
-		return "", err
+		return schema.GroupVersion{}, err
 	}
 	foundPolicyGroup := false
-	var policyGroupVersion string
 	for _, group := range groupList.Groups {
 		if group.Name == "policy" {
 			foundPolicyGroup = true
-			policyGroupVersion = group.PreferredVersion.GroupVersion
 			break
 		}
 	}
 	if !foundPolicyGroup {
-		return "", nil
+		return schema.GroupVersion{}, nil
```

Comment:
This might not be necessary, wouldn't the eviction subresource also not be here in this case?

Reply:
yeah, dropped the policy group check and just keyed off of the subresource